src/components/App/index.tsx

```diff
@@ -87,6 +94,19 @@ export default class App extends Component<Props, State> {
     this.setState({ file: undefined });
   }
 
+  @bind
+  private changeTitle() {
+    // Add the filename to the document title
+    const { file } = this.state;
+
+    // If there's a file - update title
+    if (file && file.name) {
+      document.title = `${this.defaultTitle} • ${file.name}`;
```

Comment:
I think we can put the file name first. Eg, https://www.bbc.co.uk/news/uk-politics-46269757 - the headline is in the title before "BBC news".